Update COOP/COEP article (#8239)
GoogleChrome/web.dev: src/site/content/en/blog/coop-coep/index.md

    * Use the `crossorigin` attribute in the loading HTML tag if the resource is
      served with CORS. (For example, `<img src="***" crossorigin>`.)
    * Ask the owner of the resource to support either CORS or CORP.
* For iframes, follow the same principles above and set the
  `Cross-Origin-Resource-Policy: cross-origin` (or `same-site`, `same-origin`
  depending on the context).

> * For a script loaded via a `WebWorker`, because it must be served from
>   same-origin, you don't need a CORP or CORS headers.

Reviewer: ```suggestion
* Scripts loaded with a `WebWorker` must be served from same-origin,
  so you don't need a CORP or CORS headers.
```

> * From a document or a worker served with `COEP: require-corp`, cross-origin
>   subresources loaded without CORS must set `Cross-Origin-Resource-Policy:
>   cross-origin` header to opt-in being embedded. For example, this applies to
>   `<script>`, `importScripts`, `<link>`, `<video>`, `<iframe>`, etc.

Reviewer: ```suggestion
* For a document or a worker served with `COEP: require-corp`, cross-origin
  subresources loaded without CORS must set the `Cross-Origin-Resource-Policy:
  cross-origin` header to opt into being embedded. For example, this applies to
  `<script>`, `importScripts`, `<link>`, `<video>`, `<iframe>`, etc.
```
